contract/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+========
+Contract
+========
+
+ * This module recover contracts management with recurring invoicing functions.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Sales -> Contracts and select or create a new contract.
+#. Check *Generate recurring invoices automatically*.
+#. Fill fields and add new lines.
+#. To view discount field set *Discount on lines* in user access rights.
+#. A cron is created with daily interval, but if you are in debug mode can
+   click on *Create invoices* to force this action.
+
```

Comment:
Mention here about the possibility of writing the markers, and the button for showing the created invoices.